package/yast2.changes

```diff
@@ -1,4 +1,17 @@
 -------------------------------------------------------------------
+Wed Nov  5 09:33:08 UTC 2014 - mfilka@suse.com
+
+- bnc#900383
+  - ported several fixes for device renaming
+  - added net device type detection based on sysfs
+  - fixed type detection workflow
+  - fixed device type detection when commiting new device into
+    NetworkInterfaces' cache.
```

Comment:
Do include the original Bugzilla reference, [bnc#809053](https://bugzilla.suse.com/show_bug.cgi?id=809053).